bin/testEnvLocalK8S.sh

```diff
@@ -128,49 +133,82 @@ function startLocalApiserver() {
     # make sure apiserver is actually alive
     kill -0 $(cat $LOG_DIR/apiserver.pid)
 
-    echo "Started local etcd and apiserver !"
+    # Really need to make sure that API Server is up before proceed further
```

Comment:
Well, it's a separate call because the tests scripts (at least in circle) can do some other work in parallel, while this is starting.

Reply:
However the startup is now fast enough - so maybe it doesn't matter.

Reply:
@costinm actually I added this function because I hit a failure scenario when I was trying to create a configmap but API was not ready for it. I noticed at least 3 seconds delay before API server is ready to accept requests.

Reply:
Maybe we can leave it the way it was - start, do other things (build docker images in case of CI), waitForApiserver, apply config maps.

But we can refine it - 3 seconds is not that bad.

Reply:
Not waiting for a reliable indication that api server is up and running can potentially introduced flakiness to tests. IMHO we should relay on confirmed state before moving forward.